compiler/luci/import/include/luci/Import/CircleReader.h

```diff
@@ -122,8 +122,8 @@ class CircleReader
   bool select_subgraph(uint32_t subgraph);
 
 private:
-  const circle::Model *_native_model{nullptr};
-  const circle::SubGraph *_native_subgraph{nullptr};
+  const circle::Model *_model{nullptr};
+  const circle::SubGraph *_current_subgraph{nullptr};
```

Comment:
`current_subgraph` field removed in #8011, let's return this name and give it to this member!